Pod/Classes/Logo/UIView+Logo.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIView+Logo.swift
+//  Pods
+//
+//  Created by Derek Ostrander on 7/18/16.
+//
+//
+
+import UIKit
+
+public extension UIView {
```

Comment:
#? as it's returning an UIImageView, does it make more sense to put it in an extension of UIImageView